### PR TITLE
Create and use express middleware adapter for @portrait endpoint

### PR DESCRIPTION
**Pull request by instification**
Currently portraits are fetched using `flattenToBackendURL`, which is used in the `<img src` tag.

This is problematic when using `internalApiPath` as it then asks the browser to fetch the image from a non-accessible url. For places where the avatar is rendered directly from the SSR code it also doesn't get updated (e.g. comments that display from the ssr and don't trigger a backend call).

This PR creates a simple middleware adapter, the same as for images, which proxies (with auth) to the backend and returns the portrait.

It then uses `flattenToAppUrl` where portraits are used, namely personal tools and comments.

### CODE REVIEW

**Review by tiberiuichim — APPROVED** [review]

File: src/express-middleware/portrait.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      // Just forward the headers that we need
+      HEADERS.forEach((header) => {
+        if (resource.headers[header]) {
+          res.set(header, resource.headers[header]);
+        }
+      });
+      res.send(resource.body);
+    })
+    .catch(next);
+}
```
Comment (davisagli) [inline src/express-middleware/portrait.js:17]: @instification Would it make sense to reuse imageMiddleware here instead of duplicating the code?

**Review by sneridagh — APPROVED** [review]
LGTM, waiting for resolving David's question.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,10 @@
 
 ### Feature
 
+- Added link integrity potential breakage warning message when deleting a referenced page @danielamormocea
+- Added new components & interfaces for content-rules `Rules` control in Volto. Rules management in both controlpanel and object view. @andreiggr
+- Added portrait middleware adapter. @instification
```
Comment (davisagli) [inline CHANGELOG.md:11]: @instification Careful, it looks like you picked up some unrelated changelog entries while merging/rebasing

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,10 @@
 
 ### Feature
 
+- Added link integrity potential breakage warning message when deleting a referenced page @danielamormocea
+- Added new components & interfaces for content-rules `Rules` control in Volto. Rules management in both controlpanel and object view. @andreiggr
```
Comment (davisagli) [inline CHANGELOG.md:10]:
```suggestion
```

File: CHANGELOG.md (hunk 2 above)
Comment (davisagli) [reply CHANGELOG.md:9]: I fixed it.

**Review by davisagli — APPROVED** [review]

**Pull request merged**